src/hubspot-provider.ts

```diff
@@ -0,0 +1,101 @@
+const Pkg = require('../package.json');
+
+const Hubspot = require('@hubspot/api-client')
+
+
+type HubspotProviderOptions = {}
+
+function HubspotProvider(this: any, options : HubspotProviderOptions) {
+  const seneca:any = this
+
+  const entityBuilder = this.export('provider/entityBuilder')
+
+  seneca.message('sys:provider,provider:hubspot,get:info', get_info)
+  async function get_info(this: any, _msg: any) {
+    return {
+      ok: true,
+      name: 'hubspot',
+      version: Pkg.version,
+      sdk: {
+        name: 'hubspot',
+	version: Pkg.dependencies['hubspot'],
+      }
+    }
+  }
+ entityBuilder(this, {
+   provider: {
+     name: 'hubspot'
+   },
+   entity: {
+     company: {
+       cmd: {
+         list: {
+	     action: async function(this: any, entize: any, msg: any) {
+	       const limit = msg.q.limit || 10 // The maximum number of results to display per page
+	       let res = (await this.shared.sdk.crm.companies.basicApi.getPage(limit)).results
+	       let list = res.map((data) => entize(data))
+	       return list
+	     }
+	 },
+
+	 load: {
+	   action: async function(this: any, entize: any, msg: any) {
+	     let idParts = msg.q.fields$
+	     let id = msg.q.id
+	     let obj = await this.shared.sdk.crm.companies.basicApi.getById(id, idParts) // docs for the usage: https://developers.hubspot.com/docs/api/crm/companies
+	     return entize(obj)
+	   }
+	 },
+
+	 save: {
+	   action: async function(this: any, entize: any, msg: any) {
+	     let ent = msg.ent
+	     let id = ent.id
+	     let desc = ent.properties.description
+	     let obj
+	     try{
```

Comment:
missing a space - just run https://prettier.io and all these formatting problems go away!